### PR TITLE
Collect common links under custom file

### LINKED ISSUE #70: Collect usual hyperlinks in a common links.rst
By creating a `links.rst` file and taking advantage of the `.. include:: path/to/file.rst` directive, it would be possible to reduce common links across different pages. This `links.rst` file should be included at the end of the `doc/source/index.rst` file to ensure that all pages have access to the links.

### PR DESCRIPTION
**Pull request by jorgepiloto**
Resolves #70.

### CODE REVIEW

File: doc/source/coding_style/flake8.rst [hunk 1]
```diff
@@ -2,12 +2,10 @@
...
-This topic describes the use of `Flake8`_ for `PEP8`_ style
+This topic describes the use of `flake8`_ for `PEP 8`_ style
 enforcement and the minimum standards expected. PyAnsys libraries
 are expected to be consistent with these guidelines.
 
-.. _PEP8: https://www.python.org/dev/peps/pep-0008/
-
 Flake8
 ~~~~~~
 `Flake8`_ is a Python tool for enforcing code styling. It is a wrapper
```
Comment (MaxJPRey) [inline doc/source/coding_style/flake8.rst:11]: @jorgepiloto I am assuming that the capital letter "F" is breaking the link and consequently preventing the documentation to build properly.
Indeed this link is not referenced anymore.
```suggestion
`flake8`_ is a Python tool for enforcing code styling. It is a wrapper
```

**Review by MaxJPRey — APPROVED** [review]

File: doc/source/coding_style/flake8.rst (hunk 1 above)
Comment (jorgepiloto) [reply doc/source/coding_style/flake8.rst:11]: Thanks for the feedback, @MaxJPRey. I still need to fix various hyperlinks and cross references. Wanted to do this before keep enhancing the current docs layout.

File: doc/source/abstractions/data-transfer.rst [hunk 2]
```diff
@@ -109,16 +109,5 @@ interface that can efficiently exchange a wide variety of data formats and
...
-.. _Anaconda Distribution: https://www.anaconda.com/products/individual
-.. _REST: https://en.wikipedia.org/wiki/Representational_state_transfer
-.. _PyAEDT: https://github.com/pyansys/PyAEDT
-.. _PyMAPDL: https://github.com/pyansys/pymapdl
-.. _pymapdl: https://github.com/pyansys/pymapdl
-.. _Style Guide for Python Code (PEP8): https://www.python.org/dev/peps/pep-0008
-.. _grpc_chunk_stream_demo: https://github.com/pyansys/grpc_chunk_stream_demo
-.. _numpydoc: https://numpydoc.readthedocs.io/en/latest/format.html
-.. _Namespace Packages: https://packaging.python.org/guides/packaging-namespace-packages/
+.. LINKS AND REFERENCES
```
[the remark this answers is not in the document]
Comment (PipKat) [reply doc/source/abstractions/data-transfer.rst:112]: @jorgepiloto If it's a code comment, it doesn't matter. It's acceptable as is.

**Comment by jorgepiloto** [comment]
This is likely to be introduced by some "infinite-loop" when solving links. This works fine in the HTML builder but not sure if will do so for the PDF.

**Comment by jorgepiloto** [comment]
This guide might be an alternate approach to this: https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#setup-external-links

File: doc/source/abstractions/app-interface.rst [hunk 3]
```diff
@@ -106,8 +106,9 @@ the model setup object.  There's no reason why a user needs direct
 access to ``_omodelsetup``, which is why it's protected in the
 ``Hfss`` class. Additionally, calling the method is simplified by
 providing (and documenting) the defaults using keyword arguments and
-placing them into the ``vars`` list, all while following the `Style
-Guide for Python Code (PEP8)`_.
+placing them into the ``vars`` list, all while following the Style
+Guide for Python Code, defined in `PEP 8`_.
```
Comment (PipKat) [inline doc/source/abstractions/app-interface.rst:110]:
```suggestion
placing them into the ``vars`` list, all while following the style
guidelines for Python code that are defined in `PEP 8`_.
```

File: doc/source/abstractions/data-transfer.rst [hunk 4]
```diff
@@ -3,10 +3,10 @@ Data Transfer
 
 Abstracted APIs should attempt to hide the implementation details of
 the remote or local API in a well organized data model.  This data
-model should avoid returning raw JSON files, gRPC messages, SWIG objects,
+model should avoid returning raw JSON files, gRPC messages, `SWIG`_ objects,
 or .NET objects. It should preferably return standard Python objects
```
Comment (PipKat) [inline doc/source/abstractions/data-transfer.rst:7]:
```suggestion
and .NET objects. It should preferably return standard Python objects
```

File: doc/source/coding_style/beyond_pep8.rst [hunk 5]
```diff
@@ -1,15 +1,15 @@
 Beyond PEP8
-###########
+===========
 This topic describes any delineations, clarifications, or additional procedures above and
```
Comment (PipKat) [inline doc/source/coding_style/beyond_pep8.rst:3]:
```suggestion
This topic describes any delineations, clarifications, or additional procedures 
```

File: doc/source/coding_style/beyond_pep8.rst [hunk 6]
```diff
@@ -1,15 +1,15 @@
...
+===========
 This topic describes any delineations, clarifications, or additional procedures above and 
-beyond `PEP8 <https://www.python.org/dev/peps/pep-0008/>`__.
+beyond `PEP 8`_.
 
-For example, `Stack Overflow Answer <https://stackoverflow.com/questions/2536307>`_ 
-outlines deprecation best practices and a `Deprecation library <https://deprecation.readthedocs.io/>`_ 
-already exists. However, there is no official guidance regarding deprecating features 
-in an API within Python. This page seeks to clarify this issue and others.
+For example, `Stack Overflow Answer`_ outlines deprecation best practices and a
```
Comment (PipKat) [inline doc/source/coding_style/beyond_pep8.rst:6]:
```suggestion
For example, `Stack Overflow Answer`_ outlines deprecation best practices, and a
```

File: doc/source/documentation_style/deployment.rst [hunk 7]
```diff
@@ -1,14 +1,13 @@
...
-After following the documentation build strategy contained within both
-the `pyansys/template <https://github.com/pyansys/template/>`_ and
-`pyansys-sphinx-theme
-<https://github.com/pyansys/pyansys-sphinx-theme>`_ you will end up
-with html documentation files in ``doc/build/html``.  
+Documentation for the PyAnsys project is hosted via `GitHub Pages`_ Given the
+open-source nature of the project, documentation built by each PyAnsys library
+or project should be public and editable by users and the community at large.
+
+After following the documentation build strategy contained within both the
```
Comment (PipKat) [inline doc/source/documentation_style/deployment.rst:8]:
```suggestion
After following the documentation build strategy contained within both
```

**Comment by PipKat** [comment]
@jorgepiloto and @RobPasMue  In draft [PR #410](https://github.com/ansys/pyansys-dev-guide/pull/419), I moved all links with explicit target names to a central link file, I believe that we could close this PR.)

**Comment by RobPasMue** [comment]
Let's close it yeah - thanks @PipKat - I'll review that one =)

**Pull request closed without merging**